eventlog/eventlog.go

```diff
@@ -27,26 +28,30 @@ type EventLog struct {
 	DebugEnabled bool            `json:"debugEnabled"`
 }
 
+func (l *EventLog) SetDebug(debug bool) {
+	l.DebugEnabled = debug
+}
+
 func (l *EventLog) InfoEvent(message string, args ...any) {
-	l.levelLog(linfo, message, args)
+	l.levelLog(linfo, message, args...)
```

Comment:
Face<-palm. This is one of the reasons I don't generally like to work with `any`. 😅